chainercv/links/model/fpn/faster_rcnn_fpn_resnet.py

```diff
@@ -19,7 +19,8 @@ class FasterRCNNFPNResNet(FasterRCNN):
     A subclass of this class should have :obj:`_base` and :obj:`_models`.
     """
 
-    def __init__(self, n_fg_class=None, pretrained_model=None):
+    def __init__(self, n_fg_class=None, pretrained_model=None,
+                 min_size=800, max_size=1333):
```

Comment:
`min_size` and `max_size` are not used.

Reply:
That was a versioning mistake. It will be added on L39.

Reply:
That was my mistake. Thank you for fixing!